CDAlertView/Classes/CDAlertHeaderView.swift

```diff
@@ -8,8 +8,29 @@
 
 import Foundation
 
+private extension CDAlertViewType {
+    var fillColor: UIColor? {
+        switch self {
+        case .error:
+            return UIColor(red: 235/255, green: 61/255, blue: 65/255, alpha: 1)
+        case .success:
+            return UIColor(red: 65/255, green: 158/255, blue: 57/255, alpha: 1)
+        case .warning:
+            return UIColor(red: 255/255, green: 149/255, blue: 0/255, alpha: 1)
+        case .notification:
+            return UIColor(red: 27/255, green: 169/255, blue: 225/255, alpha: 1)
+        case .alarm:
+            return UIColor(red: 196/255, green: 52/255, blue: 46/255, alpha: 1)
+        case .custom(let image):
```

Comment:
I'm not using `--allow-warnings` flag on Travis. It's better to fix every warning instead of leaving it behind. And this line generates a warning `warning: immutable value 'image' was never used; consider replacing with '_' or removing it`. Could you please replace `image` variable with `_`? Then, it'll build successfully in TravisCI